src/subscription_manager/cli_command/repos.py

```diff
@@ -120,14 +115,14 @@ def _do_command(self):
             profile_action_client = ProfileActionClient()
             profile_action_client.update()
 
-        if self.options.list:
+        if self.list:
             if len(repos):
                 # TODO: Perhaps this should be abstracted out as well...?
                 def filter_repos(repo):
                     disabled_values = ['false', '0']
                     repo_enabled = repo['enabled'].lower()
-                    show_enabled = (self.options.list_enabled and repo_enabled not in disabled_values)
-                    show_disabled = (self.options.list_disabled and repo_enabled in disabled_values)
+                    show_enabled = (self.list_enabled and repo_enabled not in disabled_values)
+                    show_disabled = (self.list_disabled and repo_enabled in disabled_values)
```

Comment:
I hope we have good unit tests for all possible cases. :smiley:

Reply:
TestReposCommand has a nice set of tests for this condition.